src/status_im/ui/components/checkbox/styles.cljs

```diff
@@ -18,9 +19,11 @@
           :width         26
           :height        26}))
 
-(def check-icon
-  {:width  12
-   :height 12})
+(defstyle check-icon
+  {:width   12
+   :height  12
+   ;;TODO(goranjovic) - this is a quickfix because checkboxes are invisible on desktop
```

Comment:
@goranjovic, is it a bug in `react-native-desktop`? Would you mind adding corresponding issue to rnd repo?

Reply:
@vkjr yep, adding them all